src/deploy/osp_deployer/settings/sample_xsp_profile.ini

```diff
@@ -171,6 +171,30 @@ StorageBond1Interface1=em2
 StorageBond1Interface2=p2p2
 StorageBondInterfaceOptions=mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
 
+[Dell NFV Settings]
+
+# OVS-DPDK Settings
+# ovs_dpdk_enable supports three values
+# ovs_dpdk_enable=None: disables OVS-DPDK
+# ovs_dpdk_enable=Tenant-networks-only: Enables OVS-DPDK in Mode 1.
+# ovs_dpdk_enable=Tenant-and-External-networks: Enables OVS-DPDK in Mode 2.
+ovs_dpdk_enable=None
```

Comment:
Can we use False to indicate disabled for consistency with the other *_enable parameters?

Reply:
Yes. Changed.